test/e2e/cypress/integration/clusters_overview.js

```diff
@@ -44,5 +46,193 @@ context('Clusters Overview', () => {
         cy.get(`a:contains(${clusterID})`).should('be.visible');
       });
     });
+
+    describe('Health status for each cluster is correct', () => {
+      before(() => {
+        cy.selectChecks(
+          clusterIdByName(healthyClusterScenario.clusterName),
+          healthyClusterScenario.checks
+        );
+        cy.requestMockRunnerResult(healthyClusterScenario.result);
+        cy.requestChecksExecution(
+          clusterIdByName(healthyClusterScenario.clusterName)
+        );
+
+        cy.selectChecks(
+          clusterIdByName(unhealthyClusterScenario.clusterName),
+          healthyClusterScenario.checks
+        );
+        cy.requestMockRunnerResult(unhealthyClusterScenario.result);
+        cy.requestChecksExecution(
+          clusterIdByName(unhealthyClusterScenario.clusterName)
+        );
+      });
+
+      after(() => {
+        cy.selectChecks(
+          clusterIdByName(healthyClusterScenario.clusterName),
+          []
+        );
+
+        cy.selectChecks(
+          clusterIdByName(unhealthyClusterScenario.clusterName),
+          []
+        );
+      });
+
+      it(`should have ${healthyClusterScenario.clusterName} displaying healthy state`, () => {
+        cy.get('td')
+          .contains(healthyClusterScenario.clusterName)
+          .parent()
+          .parent()
+          .prev()
+          .get('div > svg')
+          .should('have.class', 'fill-jungle-green-500');
+      });
+
+      it(`should have ${unhealthyClusterScenario.clusterName} displaying unhealthy state`, () => {
+        cy.get('td')
+          .contains(unhealthyClusterScenario.clusterName)
+          .parent()
+          .parent()
+          .prev()
+          .get('div > svg')
+          .should('have.class', 'fill-red-500');
+      });
+    });
+
+    describe('Clusters Tagging', () => {
+      before(() => {
+        cy.get('body').then(($body) => {
```

Comment:
We already have a command for this (removing tags):
https://github.com/trento-project/web/blob/main/test/e2e/cypress/support/commands.js#L81